bitnami/mariadb-galera/README.md

```diff
@@ -322,7 +322,9 @@ In addition to these options, you can also set an external ConfigMap with all th
 
 The allowed extensions are `.sh`, `.sql` and `.sql.gz`.
 
-Take into account those scripts are executed in all the replicas, if you want to do a "one time" action like creating a database, you need to add a condition in your `.sh` script to be executed in one of the nodes, such as
+Take into account those scripts are treated differently depending on the extension. While the `.sh` scripts are executed in all the nodes; the `.sql` and `.sql.gz` scripts are only executed in the non-bootstrapped nodes. The reason behind this differentiation is that the `.sh` scripts allow adding conditions to determine what is the node running the script, while these conditions can't be set using `.sql` nor `sql.gz` files. In this way it is possible to cover different use cases depending on the needs.
```

Comment:
In this way -> This way

Reply:
wouldn't it be `scripts are only executed in the bootstrap node`?